README.md

```diff
@@ -51,7 +51,7 @@ is to install the [Node.js client](https://github.com/tldr-pages/tldr-node-clien
 which is supported by the tldr-pages project maintainers:
 
 ```sh
-npm install -g tldr
+npm install --location=global tldr
```

Comment:
Is it a [replacement](https://docs.npmjs.com/cli/v9/using-npm/config#location) for `--global`?

Reply:
According to this warning message from `npm` they have already been deprecated. Perhaps they still work for now. Not sure if that will ever change:

```npm WARN config global `--global`, `--local` are deprecated. Use `--location=global` instead.```